LTRegistratorUI/src/app/project.ts

```diff
@@ -0,0 +1,11 @@
+export class Project {
+    name: string;
+    hours: number;
+    id: number;
+
+    constructor(name:string,hours:number, id:number){
```

Comment:
нужно настроить форматирвание кода.
```suggestion
  constructor(name: string, hours: number, id: number) {
```